File: server/src/Bananagrams.Service/Dtos/Users/UserDto.cs
using Bananagrams.Service.Dtos.GameUsers;

namespace Bananagrams.Service.Dtos.Users;

public class UserDto
{
    public int Id { get; set; }
    public DateTime DateCreated { get; set; }
    public string? EmailAddress { get; set; }
    public string? FirstName { get; set; }
    public string? LastName { get; set; }
    public string? Username { get; set; }
    
    public GameUserDto[]? GameUsers { get; set; }
}

File: server.java/bananagrams/src/main/java/coe/bananagrams/dto/user/UserDto.java
package coe.bananagrams.dto.user;

import coe.bananagrams.dto.gameUser.GameUserDto;
import lombok.*;

import java.util.Date;
import java.util.List;

@Getter
@Setter
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class UserDto {
    private int id;
    private Date dateCreated;
    private String firstName;
    private String lastName;
    private String username;
}
